setup.py

```diff
@@ -63,25 +63,27 @@
     ':python_version<"3.5"': [
         'typing'
     ],
+    ':python_version<"3.4"': [
+        'enum34'
```

Comment:
Hmm? Why did you add this to dependency?

Reply:
It's used in the Sphinx source, though currently surrounded by a `try-except ImportError`. I figured we should just make it a requirement so we can remove that `try-except`. Did I misinterpret something?

Reply:
Yes, it's not dependency. It is used to inspect user's program on autodoc module when the program uses enum. So it is not requirements of Sphinx itself.

Reply:
OK, I'll add it to `tests` instead so

Reply:
Wait - we distribute `autodoc` as part of Sphinx. Therefore shouldn't we provide all the dependencies for that module with Sphinx?

Reply:
`autodoc` uses `enum` only if it is installed. It means `autodoc` imports it only if the target program uses it. So `enum34` is a requirement of target program.
In other words, it is not "requirements", but also "optional".

Reply:
OK, I'll move this to `test` so